src/frontend/src/flows/manage/index.ts

```diff
@@ -247,11 +245,20 @@ export const renderManage = async (
     // we are actually in a device registration process
     await pollForTentativeDevice(userNumber, connection);
   } else {
-    render(pageContent(userNumber, anchorInfo.devices), container);
-    init(userNumber, connection, anchorInfo.devices);
+    displayManage(userNumber, connection, anchorInfo.devices);
   }
 };
 
+export const displayManage = async (
+  userNumber: bigint,
+  connection: AuthenticatedConnection,
+  devices: DeviceData[]
+): Promise<void> => {
```

Comment:
either this function doesn't necessary have to be a promise or `render` or `init` need to be awaited.

randomly noticed it, just in case it can be useful.

Reply:
Good catch, let me check!

Reply:
https://github.com/dfinity/internet-identity/pull/839